packages/hierarchy-browser/src/CommunityCard/CommunityCard.tsx

```diff
@@ -22,39 +21,29 @@ import { CommunityOverview } from './CommunityOverview'
 import { CommunityTable } from './CommunityTable'
 
 export interface ICommunityCardProps {
-	community: ICommunityDetail
 	maxSize: number
 	maxLevel: number
 	level: number
-	hierachyDataProvider: HierarchyDataProvider
```

Comment:
If this props interface is public, then this looks like a major version change since we're changing the prop name and removing a prop.

Reply:
The only public interface is the HierarchyBrowser Component and I don't believe the props for that changed. I just added data management from the parent level by instantiating  each Community data provider in the parent class (HierarchyBrowser) and passed to the child (CommunityCard). Previously this had been instantiated in the CommunityCard.